src/Kafka/KafkaInstances/components/InstancesTable.tsx

```diff
@@ -202,21 +202,27 @@ export const InstancesTable = <T extends KafkaInstance>({
               },
               {
                 title: t("table.actions.connection"),
-                onClick: () => onConnection(row),
+                ...(row.status === "suspended"
+                  ? {
+                      isDisabled: true,
+                    }
+                  : {
+                      onClick: () => onConnection(row),
+                    }),
               },
               {
                 isSeparator: true,
               },
               {
                 title: t("table.actions.change-owner"),
-                ...(!changeOwnerEnabled
+                ...(!changeOwnerEnabled || row.status === "suspended"
```

Comment:
@riccardo-forina, should I need to handle the logic only related to suspended in the `KafkaInstance` Component?

Reply:
Yeah. Here we should have only `true` and `false`, coming from somewhere (a callback as it is here). It's in the callback that we should have the logic that checks for the suspended status.